service/src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResourceService.java

```diff
@@ -123,6 +129,7 @@ public ControlledAzureDiskResource createDisk(
       ApiAzureDiskCreationParameters creationParameters,
       ControlledResourceIamRole privateResourceIamRole,
       AuthenticatedUserRequest userRequest) {
+    features.azureEnabledCheck();
```

Comment:
what does `features.azureEnabledCheck()` do?

Reply:
It checks that azure is enabled and throws an exception if it is not enabled.